example/bunker_small_flatroof.py

```diff
@@ -0,0 +1,52 @@
+import cadquery as cq
```

Comment:
At the top of the file please add a license, throughout the project I'm using apache2 (same as the cadquery library itself, I'm hoping the ensures compatibility). Hopefully your on board with that?! The main point is I don't limit what you can do with your generated creations, even for commercial uses I do not limit it. 

It looks like I was not very good about adding licenses to files. I will fix that problem.

Anyways, I do not intend to change the license from Apache 2 to something else but if that should ever occur than I would need your permission to do so. 
``` python 
# Copyright 2023 <your name>
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#    http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
```

Reply:
The projects I usually work on assume that all files in a project are bound by the root level licence, so that is what I was working too (although most open code I work on is BSD licenced, so it probably doesn't matter).
 
Feel free to add whatever licence you want though 😁